internal/locate/region_request.go

```diff
@@ -471,11 +483,11 @@ func (state *tryFollower) next(bo *retry.Backoffer, selector *replicaSelector) (
 	filterReplicas := func(fn func(*replica) bool) (AccessIndex, *replica) {
 		for i := 0; i < len(selector.replicas); i++ {
 			idx := AccessIndex((int(state.lastIdx) + i) % len(selector.replicas))
+			selectReplica := selector.replicas[idx]
+			hasDeadlineExceededErr = hasDeadlineExceededErr || selectReplica.deadlineErrUsingConfTimeout
```

Comment:
For fix #1170

Reply:
Why moving the two lines? They seem to have the same effect.